test/test_dataset.py

```diff
@@ -28,14 +28,13 @@ def test_open_grid_only(self):
         ux_ds3 = ux.open_dataset(uds3_name)
 
         # get node names for each grid object
```

Comment:
This comment ("node names" ) does no longer make sense (the actual data, not the names, are being initialized now). It can just be removed, and below three variables could also be removed and `ux_ds1.Mesh2_node_` could be directly used in assertion lines.